theme/static/css/hydroshare_core.css

```diff
@@ -4256,6 +4273,22 @@ div.fb-upload-caption a {
 	font-size: 18px;
 }
 
+.fb-view-list .fb-stack {
+	width: 18px;
+	height: 18px;
+	margin-right: 10px;
+	margin-left: 10px;
+}
+
+.fb-view-list .fb-stack-database .fa-database {
+	font-size: 10px;
+    top: -4px;
```

Comment:
In this case it is needed because each one is specific for each view mode (list or grid) which use different font sizes.